пр6.cpp

```diff
@@ -0,0 +1,34 @@
+#include<bits/stdc++.h>
+using namespace std;
+ 	double x, eps, y;
+float my(float x){
+	float z, q = x;
+	int n = 1;
+	int p = 1;
+	while(fabs(z) > eps){
+		q = pow(x,2*n-1)/(2*n-1);
+		cout << q << " ";
+		if(p==1){
+			z = z + q;
+			p = 2;
+		}
+		else{
+			z = z - q;
+			p=1;
+		}
+		//z +=q;

```

Comment:
```suggestion
		z += n % 2 ? -q : q;
```

Тогда не нужен весь вышестоящий `if` и переменная `p`.

Reply:
@nechaido Виконав